### ISSUE TITLE
Can you share the Loss code？

### ISSUE THREAD

**Issue by GuCheng123** [imransai/TWISE#1]
Good work! However, I see that your training code is not completely open source, especially the loss part of your paper. Could you share the loss calculation code process ? Thank you for sharing！

**Comment by leifengsoul** [comment]
Intersting job. looking forward to the training code.

**Comment by mac137** [comment]
Great publication. I also look forward to the code!
